src/django/api/models/facility/facility_index.py

```diff
@@ -54,40 +54,6 @@ class FacilityIndex(models.Model):
         db_index=True,
         default=list,
         help_text='The related approved claim facilities.'))
-    ppe_product_types = postgres.ArrayField(
```

Comment:
To properly remove these fields you should remove them via migration as well not only in the model file.